### ISSUE TITLE
Add a tutorial we made to your repo?

### ISSUE THREAD

**Issue by tpronk** [BrainardLab/mQUESTPlus#25]
Daiichiro Kuroki and I are developing a JavaScript port of QUEST+ named [jsQuestPlus](https://github.com/kurokida/jsQuestPlus). Daiichiro made a [tutorial](https://kurokida.github.io/jsQuestPlus/fitting.html#arrange-the-data) on how to import the output of jsQuestPlus into MATLAB and perform further analyses with your library (mQUESTPlus).

Given that we essentially wrote a little tutorial for mQUESTPlus, it's perhaps nice to move this tutorial to your repo instead of hours. How would you feel about this?

**Comment by DavidBrainard** [comment]
Yes, I should be able to merge this into the mQUESTPlus repo.  Will try to get to it next few weeks.

**Comment by kurokida** [comment]
We greatly appreciate your cooperation. mQUESTPlus was very helpful in creating jsQuestPlus.

**Comment by DavidBrainard** [comment]
Hi,

I took a look at this this morning.  I thought the thing to do was add run_qpFit.m to the contributed folder of mQUESTPlus, along with mydata.csv.  But when I run the demo, I don't get the same plot as in your html tutorial.  The plot I get is attached here.  This seems to match the data shown in the html, but not the data shown in your plot.  The fit seems reasonable given the data, but the example isn't as nice as the one in the html.  I didn't spend a lot of time with this, but perhaps you can clarify.

1. Does adding these two files to the contributed folder with pointers to your repository and html seem right?
2. Is there a different data file that will produce the plot in the html (but note that the tabular data in the html seem to match the current file)?

**Comment by kurokida** [comment]
Thank you for reviewing. Actually, to make it easier to explain in the tutorial, I limited the number of trials for run_qpFit.m to 10 trials which is too few for fitting. The plot shown in the tutorial was obtained using another 32 trials data.

When I created the tutorial, I thought this explanation was easy to understand, but in light of your point, I have changed the run_qpFit.m to include 32 trials. I have also made slight changes to [the tutorial content](https://kurokida.github.io/jsQuestPlus/fitting.html). Sorry for the confusion.

I agree with the inclusion of both run_qpFit.m and mydata.csv in the contributed folder, but could you wait for @tpronk 's reply?

**Comment by DavidBrainard** [comment]
@tpronk Your thoughts on the above?

**Comment by tpronk** [comment]
Sounds good to me :)
